docs/user/intro.md

```diff
@@ -23,6 +23,27 @@ import OcLogoLilac from '/static/img/oc-logo-lilac.svg';
 Here you will find all the important information to successfully take the first steps in OpenCloud, to be able to solve
 technical problems flexibly yourself and to be able to answer frequently asked questions.
 
+## What is OpenCloud?
+
+OpenCloud is the file sharing and collaboration solution of the Heinlein Group.
```

Comment:
the currently first sentence says "Welcome to OpenCloud, your self-hosted file sharing platform: Secure, simple and reliable. " so this info is already given.

That there is a Heinlein Group is noise for someone who seeks help.